proxy-server/proxy-server.go

```diff
@@ -269,15 +271,16 @@ func handleRequestAndRedirect(res http.ResponseWriter, req *http.Request) {
 
 	if len(tmpTargetURL) == 0 {
 		configMutex.Lock()
-		tmp, ok := targetMap[extractTargetURLKey(req.Host)]
+		targetKey := extractTargetURLKey(req.Host)
+		tmp, ok := targetMap[targetKey]
 		configMutex.Unlock()
 		if ok {
 			tmpTargetURL = tmp
 			if debug {
 				log.Printf("Debug: target url is %s", tmp)
 			}
 		} else {
-			log.Printf("Unknown target url for %s", req.Host)
+			log.Printf("Unknown target url for %s", targetKey)
```

Comment:
I put there full url for a purpose. if extractTargetURLKey works incorrectly it would be hard to know, it would look like we have no entry.  If we got full .Host we can deduct key ourselves and see if it was done correctly.

Reply:
That's fair - I put this in when I was debugging a bizarre thing that was being caused by saturn. I'll revert this.